src/Avalonia.Themes.Fluent/Controls/TextBox.xaml

```diff
@@ -232,7 +234,7 @@
   <!-- TextBox.Button Normal State -->
   <Style Selector="Button.textBoxClearButton, ToggleButton.passwordBoxRevealButton">
     <Setter Property="MinWidth" Value="34" />
-    <Setter Property="Width" Value="{Binding $self.Bounds.Height}"/>
+    <Setter Property="Width" Value="{ReflectionBinding $self.Bounds.Height}"/>
```

Comment:
This should work after my recent changes.

Reply:
There is some problem with Bounds,it cant be used with CompiledBindings

Reply:
Can you create an issue with this problem? Would be nice to investigate it.

Reply:
turns out it works actually, just writes compile-time error message in log, but seems to be a false alarm because it compiles and works. But still doesn't work with TemplateBinding,if I will rewrite following code to use TemplateBinding it will fail
![image](https://user-images.githubusercontent.com/53405089/159025518-3c5da02f-c5cc-4f38-a076-f41e7a58905b.png)
![image](https://user-images.githubusercontent.com/53405089/159025523-f34f89a2-a794-4bf8-9966-5cdc9275555f.png)

Reply:
I didn’t see any errors like that - perhaps extension is causing this?

Reply:
no idea tbh(

Reply:
Also I’m not sure if templated parent binding can work with compiled bindings at all. Since I think type in this case is not known.